mlir/test/Conversion/MIGraphXToTosa/mixr-to-tosa-ops.mlir

```diff
@@ -22,6 +22,17 @@ module  {
      return %0 : tensor<32x64xf32>
   }
 
+  // CHECK-LABEL: func.func @matmul_broadcast
+  func.func @matmul_broadcast(%arg0: tensor<64x64x2304xf16>, %arg1: tensor<64x64x768xf16>, %arg2: tensor<1x768x2304xf16>) -> tensor<64x64x2304xf16> attributes {arch = "gfx90a:sramecc+:xnack-", kernel = "mixr"} {
+    %0 = migraphx.multibroadcast(%arg2) {out_dyn_dims = [], out_lens = [64, 768, 2304]} : (tensor<1x768x2304xf16>) -> tensor<64x768x2304xf16>
+    // CHECK: [[RESHAPE1:%.+]] = tosa.reshape
```

Comment:
I think it would be better have the shapes in the test to understand this change for future us...

Reply:
done